### DISCUSSION TITLE
Progress on Mortgage Calculator

### DISCUSSION THREAD

**Discussion by gh44922023** [gh44922023/Tafe230221#9] [General]
@BrodieBrown1998 
Just checking on the progress of "Mortgage Calculator" Development.
Any issues to report?
Any screenshots to share?

**Comment by BrodieBrown1998** [comment]
I've got the menu, and the design of the mortgage calculator done, just need Stephen's help with the equation to calculate mortgage repayment and I'll pushing to GitHub.
![Menu Screenshot](https://github.com/gh44922023/Tafe230221/assets/141377278/0e169220-65a7-4a8f-a8e9-6418056220a8)
![Mortgage Screenshot](https://github.com/gh44922023/Tafe230221/assets/141377278/f5be26c6-3678-44d6-927c-402aeef88916)

**Comment by gh44922023** [comment]
Good Work Brodie!  Job well done!

**Comment by jeffrey-tafe** [comment]
Looks good mate.
